src/static_directory_server.rs

```diff
@@ -31,25 +31,27 @@ impl StaticDirectoryServer {
         // port is very important because it enables creating multiple instances at the same time.
         // We need this to be able to run tests in parallel.
         let addr = SocketAddr::new([127, 0, 0, 1].into(), 0);
-        let server = axum::Server::bind(&addr).serve(app.into_make_service());
```

Comment:
`axum::Server` was removed in 0.7.0. The changelog suggests using `axum::serve`: https://github.com/tokio-rs/axum/blob/main/axum/CHANGELOG.md#070-27-november-2023.